modules/json/src/parsers/convert-json-expression.js

```diff
@@ -0,0 +1,74 @@
+import {get} from '../utils/get';
```

Comment:
Maybe rename this file and function to `parse-string-expression` (rather than `convert-json-expression`).

My thinking is that the expressions we parse here are not JSON syntax but an arbitrary syntax we are defining, almost as an "extension" of JSON.